packages/interface/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "cw-orch-polytone"
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cw-orch = { version="0.16.4", features=["daemon"] }
+
+polytone-note = { version = "1.0.0", path = "../../contracts/main/note" }
+polytone-proxy = { version = "1.0.0", path = "../../contracts/main/proxy" }
+polytone-voice = { version = "1.0.0", path = "../../contracts/main/voice" }
```

Comment:
Could we use the workspace for these deps?

Reply:
Done in https://github.com/DA0-DA0/polytone/pull/54/commits/61e81bf079eb689ccaa5fde51a238ef2497ccbcb